packages/api-form-builder/src/plugins/validators/dateGte.ts

```diff
@@ -0,0 +1,31 @@
+import { validation } from "@webiny/validation";
+import { FbFormFieldValidatorPlugin } from "~/types";
+
+const plugin: FbFormFieldValidatorPlugin = {
+    type: "fb-form-field-validator",
+    name: "form-field-validator-date-gte",
+    validator: {
+        name: "dateGte",
+        validate: async (value, validator) => {
+            // With this regex we check whether our value is time (11:59) and not a date
```

Comment:
Let's be grammatically correct when writing comments. For example, here we're missing dots at the end of each sentence. Sorry to nitpick, but tiny details like these matter to us.